helper_test.go

```diff
@@ -2025,6 +2027,59 @@ func createProject(t *testing.T, client *Client, org *Organization) (*Project, f
 	}
 }
 
+func createTarGzipArchive(t *testing.T, files []string, outputPath string) {
+	out, err := os.Create(outputPath)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer out.Close()
+
+	gw := gzip.NewWriter(out)
+	defer gw.Close()
+
+	tw := tar.NewWriter(gw)
+	defer tw.Close()
+
+	if len(files) == 0 {
```

Comment:
since the createTar... is terminated if files are empty, it might be best to move this statement to lines 2031 so that the `os` doesn't bother creating the above path only to discard it in empty cases.